### PR TITLE
Prepare v0.5.2 release

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+
 terraform {
   required_version = ">= 0.12.20, < 2.0"
 
   required_providers {
-    aws = ">= 2.0, < 4.0"
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 2.0, < 4.0"
+    }
```
Comment (mariux) [inline versions.tf:12]: is this supported by terraform 0.12?

**Review by mariux — CHANGES_REQUESTED** [review]

File: versions.tf (hunk 1 above)
Comment (Hanymin) [reply versions.tf:12]: Terraform v0.12.26 will accept syntax like the above but will understand it in the same way as the following v0.12-style syntax:

terraform {
  required_providers {
    aws = "~> 1.0"
  }
}

Comment (soerenmartius) [reply versions.tf:12]: > is this supported by terraform 0.12?

ah shit.. will change

**Comment by soerenmartius** [comment]
r4r once again @mariux

**Review by mariux — APPROVED** [review]
lgtm

**Comment by samen93** [comment]
@soerenmartius  i don't understand the change here but looks good to me

**Comment by soerenmartius** [comment]
> @soerenmartius i don't understand the change here but looks good to me

it's obvious no? we prepare a release in `CHANGELOG.md`

**Pull request merged**